### PR TITLE
Scenario Deploy Azure Monitor on AKS on Azure Stack HCI

### PR DESCRIPTION
**Pull request by lanicolas**
Automation for deploying Azure Monitor on AKS HCI using extensions

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks_stack_hci/aks_hci_monitor/aks_hci_monitor_extension/_index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource Insights](./02.png)
+
+    ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource Insights](./03.png)
+
+* Edit the environment variables in the [PowerShell script](https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_k8s_jumpstart/aks_stack_hci/monitor_extension/azure_monitor_k8s_extension.ps1) to match your environment parameters. After that run it using the ```.\azure_monitor_k8s_extension.ps1``` command.
+
+    ![Screenshot showing environment variables in script](./04.png)
+
+    > **Note: The extra dot is due to the shell script has an *export* function and needs to have the vars exported in the same shell session as the rest of the commands.**
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks_stack_hci/aks_hci_monitor/aks_hci_monitor_extension/_index.md:80]: This is only in bash

**Review by likamrat — CHANGES_REQUESTED** [review]
Minor Note removal

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks_stack_hci/aks_hci_monitor/aks_hci_monitor_extension/_index.md (hunk 1 above)
Comment (lanicolas) [reply docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks_stack_hci/aks_hci_monitor/aks_hci_monitor_extension/_index.md]: True, fixed

**Review by likamrat — APPROVED** [review]
LGTM!

**Pull request merged**